<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>1240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54,54,54);</string>
  </property>
  <widget class="QLabel" name="DM_label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>30</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:30pt;color:rgb(255, 255, 255)
</string>
   </property>
   <property name="text">
    <string>DM</string>
   </property>
  </widget>
  <widget class="QLabel" name="DM_label_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>170</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Date Manager</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_button">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>510</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 247, 246);</string>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>start_clicked()</slot>
 </slots>
</ui>
